audit/projects/kubernetes-public/buckets/k8s-metrics/metadata.txt

```diff
@@ -0,0 +1,18 @@
+gs://k8s-metrics/ :
```

Comment:
Expected, this is Expected, this is https://github.com/kubernetes/k8s.io/pull/2451